docs/articles_en/openvino-workflow/model-optimization-guide/compressing-models-during-training/quantization-aware-training-pytorch.rst

```diff
@@ -0,0 +1,114 @@
+Quantization-aware Training (QAT) with PyTorch
+===============================================
+
+Here, we provide the steps that are required to integrate QAT from NNCF into the training script written with
+PyTorch:
+
+1. Import NNCF API
```

Comment:
Steps 1-3 are exactly the same as in http://openvino-doc.iotg.sclab.intel.com/nncf-docs/openvino-workflow/model-optimization-guide/quantizing-models-post-training/basic-quantization-flow.html. Can we simply say that the initial step is to get the quantized PyTorch model using PTQ and refer to that page?

Reply:
Done. I've added a small snipped to define the `quantized_model` value for reference, please check